newIDE/app/src/InstancesEditor/TileSetVisualizer.js

```diff
@@ -0,0 +1,640 @@
+// @flow
+
+import * as React from 'react';
+import { createStyles, makeStyles } from '@material-ui/core/styles';
+import { Column, Line } from '../UI/Grid';
+import { CorsAwareImage } from '../UI/CorsAwareImage';
+import ResourcesLoader from '../ResourcesLoader';
+import Erase from '../UI/CustomSvgIcons/Erase';
+import Brush from '../UI/CustomSvgIcons/Brush';
+import IconButton from '../UI/IconButton';
+import { LineStackLayout } from '../UI/Layout';
+import FlipHorizontal from '../UI/CustomSvgIcons/FlipHorizontal';
+import FlipVertical from '../UI/CustomSvgIcons/FlipVertical';
+import useForceUpdate from '../Utils/UseForceUpdate';
+
+const styles = {
+  tileContainer: { flex: 1, position: 'relative', display: 'flex' },
+  atlasImage: { flex: 1, imageRendering: 'pixelated' },
+  icon: { fontSize: 18 },
+};
+
+const useStylesForTile = (highlighted: boolean) =>
+  makeStyles(theme =>
+    createStyles({
+      tile: {
+        position: 'absolute',
+        boxSizing: 'border-box',
+        border: highlighted ? '2px solid red' : undefined,
+        '&:hover': {
+          border: highlighted ? '2px solid pink' : '1px solid white',
```

Comment:
maybe double check it looks ok on dark and light theme

Reply:
Thanks, I changed the color depending on the palette type